### PR TITLE
Added Marketing CTA to readme

### PR DESCRIPTION
**Pull request by taras**
This PR adds a call to action for someone who works for a company to reach out to us if they'd like to work with us. 

image

### CODE REVIEW

**Comment by taras** [comment]
@cowboyd how about just

> We are looking for partner companies to help us evolve Microstates. If you are interested in supporting Microstates development by hiring Frontside, please reach out via [email](mailto:info@frontside.io)

File: README.md [hunk 1]
```diff
@@ -891,6 +891,10 @@ class Employee extends Person {
 }
 ```
 
+## Supporting Development
+
+We are looking for partner companies to help us evolve Microstates. If you are interested in supporting Microstates development by hiring Frontside, please reach out via [email](mailto:info@frontside.io)
```
Comment (Robdel12) [inline README.md]: Should this be `info@`? Or `oss@`, `sales@`?

Comment (taras) [reply README.md]: Sales?

Comment (cowboyd) [reply README.md]: What's the call here?

Comment (Robdel12) [reply README.md]: I think any of it works. `oss@frontside.io` makes the most sense imo

**Comment by cowboyd** [comment]
@taras sounds like we just need to adjust this to `oss@frontside.io` and then we're good.

**Pull request merged**